src/invoices/InvoiceDetails/Information/Information.js

```diff
@@ -60,21 +61,21 @@ const Information = ({
           <KeyValue
             label={<FormattedMessage id="ui-invoice.invoice.details.information.paymentDue" />}
           >
-            {formatDate(paymentDue)}
+            {paymentDue ? formatDate(paymentDue) : <NoValue />}
           </KeyValue>
         </Col>
 
         <Col xs={3}>
           <KeyValue label={<FormattedMessage id="ui-invoice.invoice.details.information.terms" />}>
-            {paymentTerms}
+            {paymentTerms || <NoValue />}
           </KeyValue>
         </Col>
 
         <Col xs={3}>
           <KeyValue
             label={<FormattedMessage id="ui-invoice.invoice.details.information.approvedDate" />}
           >
-            {formatDate(approvalDate)}
+            {approvalDate ? formatDate(approvalDate) : <NoValue />}
```

Comment:
`formatDate` might be deprecated, maybe it makes sense to use `FolioFormattedDate` instead

Reply:
Actually it's the same) https://github.com/folio-org/ui-invoice/blob/master/src/common/utils/invoiceUtils.js#L14

Reply:
It's not the same in case of this particular line. It makes sense to leave `{formatDate(approvalDate)}` as is or to put FolioFormattedDate here directly